### PR TITLE
add default parameters to default rendering pipeline, add glow enable…

### PR DESCRIPTION
**Pull request by TrevorDev**
…d setter

### CODE REVIEW

File: src/PostProcess/RenderPipeline/Pipelines/babylon.defaultRenderingPipeline.ts [hunk 1]
```diff
@@ -337,15 +358,15 @@
...
+         * @param {boolean} hdr - If high dynamic range textures should be used (default: true)
+         * @param {BABYLON.Scene} scene - The scene linked to this pipeline (default: the last created scene)
+         * @param {BABYLON.Camera[]} cameras - The array of cameras that the rendering pipeline will be attached to (default: scene.cameras)
+         * @param {boolean} automaticBuild - if false, you will have to manually call prepare() to update the pipeline (default: true)
          */
-        constructor(name: string, hdr: boolean, scene: Scene, cameras?: Camera[], automaticBuild = true) {
+        constructor(name: string = "", hdr: boolean = true, scene: Scene = BABYLON.Engine.LastCreatedScene!, cameras?: Camera[], automaticBuild = true) {
             super(scene.getEngine(), name);
-            this._cameras = cameras ||  [];
+            this._cameras = cameras ||  scene.cameras;
```
Comment (TrevorDev) [inline src/PostProcess/RenderPipeline/Pipelines/babylon.defaultRenderingPipeline.ts]: This is a possible breaking change.

Comment (sebavan) [reply src/PostProcess/RenderPipeline/Pipelines/babylon.defaultRenderingPipeline.ts]: I guess it is acceptable as a change adding @deltakosh for inputs ?

**Comment by TrevorDev** [comment]
@sebavan dispose is on line 316

**Comment by sebavan** [comment]
Is this one also called when disposing the defaultrenderingpipeline itself ?

**Comment by TrevorDev** [comment]
Oops nope, added.

**Pull request merged**